src/Altinn.Platform/Altinn.Platform.Events/Events/Services/Interfaces/IEventsService.cs

```diff
@@ -0,0 +1,18 @@
+using System.Threading.Tasks;
+using Altinn.Platform.Events.Models;
+
+namespace Altinn.Platform.Events.Services.Interfaces
+{
+    /// <summary>
+    /// Something smart
```

Comment:
I guess it's time to find something smart to write here :P

Reply:
Something smart added :)